e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -52,8 +52,9 @@ Scenario('Judiciary enters hearing details and submits', async (I, caseViewPage,
   I.seeAnswerInTab(5, 'Hearing 1', '', hearingDetails[0].type.welsh);
   I.seeAnswerInTab(5, 'Hearing 1', '', hearingDetails[0].type.somethingElse);
   I.seeAnswerInTab(6, 'Hearing 1', 'Give details', hearingDetails[0].giveDetails);
-  I.seeAnswerInTab(7, 'Hearing 1', 'Judge or magistrate\'s title', hearingDetails[0].judgeTitle);
-  I.seeAnswerInTab(8, 'Hearing 1', 'Judge or magistrate\'s last name', hearingDetails[0].lastName);
+  I.seeAnswerInTab(1, 'Judge and legal advisor', 'Judge or magistrate\'s title', hearingDetails[0].judgeAndLegalAdvisor.judgeTitle);
```

Comment:
IMO `hearingDetails[0].judgeAndLegalAdvisor` should be split into `hearingDetails[0].judge` and `hearingDetails[0].legalAdvisor`.

It might be just comment for the future. Cost of refactoring it now might be too big and might be actually better to keep it consistent.

Just a note.

Reply:
Might be nice to split them out in the future, but to be consistent with our java models/ccd definitions/rest of the code I kept it together (we keep them together everywhere)